templates/scale_up_resources.j2

```diff
@@ -53,8 +53,7 @@ ms msl_SAPHana_{{ sid }}_HDB{{ instance }} rsc_SAPHana_{{ sid }}_HDB{{ instance
 {% if data.platform == "azure" %}
 
 primitive rsc_nc_{{ sid }}_HDB{{ instance }} anything \
-  params binfile="/usr/bin/nc" cmdline_options="-l -k 62503" \
-  meta resource-stickiness=0 \
+  params binfile="/usr/bin/socat" cmdline_options="-U TCP-LISTEN:62503,backlog=10,fork,reuseaddr /dev/null" \
```

Comment:
@Simranpal I think the `TCP-LISTEN` port cannot be hardcoded as the last 2 digits are the HANA instance number.
Could you confirm that?
It should be `TCP-LISTEN:625{{ instance }}`

Reply:
Hi @arbulu89 yes this seems to be the HANA probe port which is based on the instance number.

https://documentation.suse.com/sbp/all/pdf/SBP-SAP-AzureSolutionTemplates_color_en.pdf
here where hana instance is 00